src/lib/Preview.js

```diff
@@ -957,13 +967,17 @@ class Preview extends EventEmitter {
             // Bump up preview count
             this.count.error += 1;
 
+            const metrics = this.fileMetrics.done(this.count);
+
             // 'load' with { error } signifies a preview error
             this.emit('load', {
                 error,
-                metrics: this.logger.done(this.count),
+                metrics,
                 file: this.file
             });
 
+            this.logger.metric(METRIC_FILE_PREVIEW_FAIL, metrics);
```

Comment:
Do we want to duplicate this information? I thought what we are capturing now isn't "logging"

Reply:
My thoughts are that we will capture this for historical purposes, then once we break it down into multiple useful metrics (and use the new logger) we'll remove this.